Makefile

```diff
@@ -460,3 +465,15 @@ localnet-stop:
 	docker-compose down
 
 .PHONY: localnet-start localnet-stop
+
+###############################################################################
+###                                rosetta                                  ###
+###############################################################################
+# builds rosetta test data dir
+rosetta-data:
+	-docker container rm data_dir_build
```

Comment:
I think there is a typo here. Also, is this line still needed (we have it in 478 as well)

Reply:
the `-` supresses errors. required because in case the container name is taken we reclaim it.